build.gradle

```
@@ -8,7 +8,7 @@ plugins {
 }
 
 group 'com.surrealdb'
-version '0.1.0'
+version '0.1.1'
```

This wouldnt be a patch, and we would version into 0.2.0 independently when we do a release
```suggestion
version '0.1.0'
```